python-sdk/tests/benchmark/wait_for_completion.sh

```diff
@@ -0,0 +1,17 @@
+set -x  # print the shell command before being executed
+complete_status=1
+failed_status=1
+while [[ $complete_status -ne 0 ]] && [[ $failed_status -ne 0 ]]; do
+  sleep 60
+  output=$(kubectl wait --for=condition=failed job/benchmark -n benchmark --timeout=0 2>&1)
+  failed_status=$?
+  output=$(kubectl wait --for=condition=complete job/benchmark -n benchmark --timeout=0 2>&1)
+  complete_status=$?
+done
+
+kubectl delete job benchmark -n benchmark
+
+if [ $failed_status -eq 0 ]; then
+    echo "Job failed. Please check logs."
```

Comment:
What if we displayed the error from the K8s pod triggered by the job to make troubleshooting easier?

Reply:
I think we need a pod name to fetch logs, right?
how we can get the pod name here, any thoughts?